careers/settings.py

```diff
@@ -184,6 +187,7 @@
     'http://*.mozilla.net',
     'https://*.mozilla.net',
 )
+CSP_REPORT_URI = lazy(lambda: reverse('csp-violation-capture'), unicode)()
```

Comment:
I'd add a `CSP_REPORT_ONLY` here as default `False` and configurable. I'm implementing this in my csp branch for bedrock now, and I did this:

```python
CSP_REPORT_ONLY = config('CSP_REPORT_ONLY', default=False, cast=bool)
```

It will allow you to quickly disable CSP on the site while still getting the reports so you can diagnose and fix.